app/templates/components/tag-item.hbs

```diff
@@ -1 +1 @@
-{{#link-to 'tag' (query-params tag = tag.name)}}<span class="w3-tag {{tagClass}} w3-margin-bottom">{{capitalize tag.name}}</span>{{/link-to}}
+{{#link-to 'tag' tag.id}}<span class="w3-tag {{tagClass}} w3-margin-bottom">{{capitalize tag.name}}</span>{{/link-to}}
```

Comment:
@mansona Yep, this is what I expected to work. The `query-params` was an attempt to workaround the problem.